tests/loggers/test_all.py

```diff
@@ -235,11 +235,9 @@ def on_train_batch_start(self, trainer, pl_module, batch, batch_idx, dataloader_
 @pytest.mark.skipif(platform.system() == "Windows", reason="Distributed training is not supported on Windows")
 @pytest.mark.parametrize("logger_class", [
     TensorBoardLogger,
-    CometLogger,
     MLFlowLogger,
     NeptuneLogger,
     TestTubeLogger,
-    WandbLogger,
```

Comment:
@williamFalcon are these tests causing problems?

Reply:
yes... these loggers are having issues where they freeze and such. I’m honestly not sure it’s worth running some of these tests anymore if the loggers aren’t properly designed.

Reply:
oh, that's new. I use wandb logger every day non stop and no issues with PL. Can you give me some pointer, I'd like to help fix the root issue.

Reply:
in the mean time, can we comment the lines and add a todo note?

Reply:
sure. i’ll ping you on slack